test/e2e-tests.sh

```diff
@@ -124,19 +140,33 @@ function gcppubsub_teardown() {
   kubectl -n knative-eventing delete secret ${PUBSUB_SECRET_NAME}
 }
 
+# Create resources required for NATSS provisioner setup
+function natss_setup() {
```

Comment:
should this be in a separate script ?  I assume this file can become messy w/ future addons, like for Kafka.

Reply:
+1 for moving support functions to a separate file. But this can be done in a separate PR IMO.